### DISCUSSION TITLE
Deployment without any Helm installation?

### DISCUSSION THREAD

**Discussion by remoe** [fluxcd/flux2#3044] [General] [answered]
Sorry, Novice question: Is it possible to deploy without use/install helm-controller? Is Helm a hard requirement to use FluxCD 2?

**Comment by stefanprodan** [comment, marked as answer]
You can use Flux without its Helm controller, the minimum required components for `flux bootstrap` are source and kustomize controller. You can chose the controllers you want using the `--components ` flag.
